cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -21,10 +21,10 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 8Mi
+            memory: 16Mi
           requests:
```

Comment:
One question here, instead of bump the request here, why not just remove the request?